twitter/setup.py

```diff
@@ -14,11 +14,12 @@
         #'tweetstream==0.2.0',
         'ujson',
         'configobj',
-        'oauth2-utf8==1.5.170',
+        'oauth2',
         'kafka-python',
         'PyYAML>=5',
         'warctools',
-        'scandir;python_version<"3"'
+        'twitter-stream.py',
+        'scandir;python_version>"3.7"'
```

Comment:
is `scandir` necessary for Python 3.8 and up? `scandir` is Python 3 `scandir` pkg backported to Python 2, IIRC.

Reply:
Yes, we can remove that from dependancies. I will remove it and make a commit.

Reply:
Made the changes and pushed it.